### PR TITLE
Prevent css file caching of browsers

### PR DESCRIPTION
**Pull request by NolwennP**

### CODE REVIEW

File: tasks/landings.js [hunk 1]
```diff
@@ -16,6 +17,7 @@ gulp.task('landings', function() {
                     partialsDirectory: ['./src/partials']
                 }))
                 .pipe(rename(path.basename(file.path).replace(/\.handlebars$/, '.html')))
+                .pipe(revReplace({manifest: gulp.src("./tmp/rev/rev-manifest.json")}))
```
Comment (ahocquard) [inline tasks/landings.js]: your revision manifest is calculated only from the css directory. Then, you can have problem if you modify an handlebar with css, or an HTM file, but your with a css directory without any modification.

We have to calculate the manifest revision from all the directory: css + handlebar + html.

Comment (ahocquard) [inline tasks/landings.js]: Seen IRL, it's ok.

File: tasks/less.js [hunk 2]
```diff
@@ -4,11 +4,15 @@
...
 gulp.task('less', ['clean-dist'], function () {
     return gulp.src('./styles/variables.less')
         .pipe(less({
             paths: [ path.join(__dirname, 'less', 'includes') ]
         }))
-        .pipe(gulp.dest('./dist/css'));
+        .pipe(rev())
+        .pipe(gulp.dest('./dist/css'))
+        .pipe( rev.manifest() )
+        .pipe( gulp.dest('./tmp/rev'));
```
Comment (pierallard) [inline tasks/less.js]: We should install a checkstyle on day :trollface:

Comment (NolwennP) [inline tasks/less.js]: :D

**Pull request merged**